app/src/main/res/values-ar/strings.xml

```diff
@@ -0,0 +1,429 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="accounts">الحسابات</string>
+    <string name="total">المجموع: %1$s %2$s</string>
```

Comment:
1% is broken, should be %1$s

Reply:
Yes I'm aware. It's just a problem when Arabic and English characters come together. But I've tried it and it's format right although it does not look like it is.

Reply:
Tomorrows, I'll will make little changes anyways because I found some things that need to be added (like: spaces and some small text formats in arabic).

Reply:
@ILIYANGERMANOV Fixed it and some other strings. it still does not look correct here on GitHub but it is formatted correctly on android studio and on the app itself